### ISSUE TITLE
Avoid setting control mode at each control loop iteration

### ISSUE THREAD

**Issue by traversaro** [robotology-playground/WBI-Toolbox#41]
cc @francesco-romano

**Comment by traversaro** [comment]
cc @francesco-romano @DanielePucci , on this topic check also https://github.com/robotology-playground/yarp-wholebodyinterface/issues/9

**Comment by jeljaik** [comment]
It isn't completely clear to me if this issue was solved at the yarp-wholeBodyInterface level. I remember @traversaro saying that in v0.2 once it's set future calls for the same control mode are ignored. Am I correct here?

**Comment by traversaro** [comment]
Yes, that is the behaviour that I have implemented to solve this issue but I really don't like it, but solving this is low priority I guess.
